poleno/attachments/views.py

```diff
@@ -31,4 +31,4 @@ def upload(request, generic_object, download_url_func):
 def download(request, attachment):
```

Comment:
Neupdatol si `upload` funkciu, ktora vytvara `Attachment` a posiela do neho `content_type` od usera.

V zadani som pisal, ze treba grepnut celu codebase a upravit pouzitia `content_type`, ktore su zle. Napriklad do `Attachment` sa `content_type` uz nema posielat, kedze metoda `save` ho dopocita sama. Rovnako na ostatnych miestach, kde sa vyrabaju nove `Attachment`.

Grepni si celu codebase na slova `content_type` a `Attachment`, zamysli sa, ktore suvisia s touto zmenou a uprav to.